.github/workflows/pre_release.yml

```diff
@@ -0,0 +1,49 @@
+name: pre-release
+
+on:
+  create
+
+jobs:
+
+  # Make a PR to master from a new branch with changes, and delete the created one
+  update-versions:
+
+    # Only run when a release-v* branch is created, and not by drashbot
+    if: contains(github.ref, 'release-v') && !contains(github.event.sender.login, 'drashbot')
+
+    strategy:
+      matrix:
+        deno: ["1.5.1"]
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Delete Remote Release Branch
+        run: git push --delete origin ${{ github.ref }}
```

Comment:
do we want to delete the branch still?

Reply:
disregard. i was asking because i was thinking about LTS, but i think we're ok deleting the branch because the branch name for LTS branches would be our `v1.x`, `v2.x`, etc. branches. resolving this.